tests/FSharpLint.Core.Tests/Rules/Conventions/VariableLengthGreaterThanOne.fs

```diff
@@ -0,0 +1,59 @@
+module FSharpLint.Core.Tests.Rules.Conventions.VariableLengthGreaterThanOne
+
+open NUnit.Framework
+open FSharpLint.Framework.Rules
+open FSharpLint.Rules
+
+[<TestFixture>]
+type TestConventionsVariableLengthGreaterThanOne() =
+    inherit TestAstNodeRuleBase.TestAstNodeRuleBase(VariableLengthGreaterThanOne.rule)
+
+    [<Test>]
+    member this.VariableLengthMoreThanOne() =
```

Comment:
this test still doesn't have a good name, what are you testing here?

Reply:
(e.g. if you keep consistency with the other test names, this test name should include the word "Should")

Reply:
Updated.